### DISCUSSION TITLE
Accounted has been blocked,too!

### DISCUSSION THREAD

**Discussion by dust6174** [vercel/community#4208] [Account Issue] [answered]
### Question

I haven't even started using it yet..

**Comment by hungchihsueh** [comment]
same here please help !, i want my  domain back

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
